src/game-logic/reducers/harvestPlot.js

```diff
@@ -96,3 +85,40 @@ export const harvestPlot = (state, x, y) => {
     },
   }
 }
+
+function harvestWeeds(state, x, y) {
```

Comment:
if you feel strongly i can change it, but I like the "harvest" verbage here personally because it's part of the "harvestPlot" logic. and i kinda mentioned it in discord, but in my mind a more ultimate version is something that can actually call the same method on anything that is occupying the plot i.e. `plotItem.harvest()` rather than this reducer having to make a decision about how to handle what's in there. 

i am definitely going to singularize it though based on making the other stuff singular so it's "harvestWeed". like i said though, i can change it if you have a preference for "pull".

Reply:
I don't feel strongly about "pull" over "harvest," and I agree that it sets the stage for more scalable and generic semantics. Good thinking!